tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = py27
+
+[testenv]
+basepython = python
+
+deps =
+    coverage
+    flake8
+    pytest
+    mock
+    requests
+    treeherder-client
+
+commands =
+    flake8 triggerbot test buildapistats
+    coverage run --source=triggerbot -m py.test test
```

Comment:
I haven't used pytest before, but I can't tell whether this is going to work as expected. The tests we have are based on the unittest api, will tox/pytest detect and run those? A good way to test this would be to run the coverage command and see what it reports (and possibly post the results here, I can tell you whether they seem accurate given the tests we have).